crates/biome_js_analyze/tests/specs/nursery/noUndeclaredDependencies/invalid.js

```diff
@@ -0,0 +1 @@
+import "notInstalled";
```

Comment:
What happens if `notInstalled` package is a sub-dependency of `react`, and hence not included in the project's package.json?

Reply:
You'll still get the diagnostics because it's not a dependency declared in the manifest. I believe it's an anti-pattern to use an indirect dependency, because they aren't controlled by the user.

Reply:
Can we document this somewhere?

Reply:
Done